ludwig/schema/trainer.py

```diff
@@ -158,6 +159,16 @@ class ECDTrainerConfig(BaseTrainerConfig):
         ],
     )
 
+    max_batch_size: Optional[int] = schema_utils.PositiveInteger(
+        default=sys.maxsize,
```

Comment:
`sys.maxsize` may vary depending on the architecture its running on, IMO it would be better to make this a large constant.   Or it could default to None, since allow_none=True.

Reply:
See below comment.